templates/about/publicise.html

```diff
@@ -8,104 +8,138 @@
+</section>
+
+<section class="p-strip is-shallow">
+  <div class="u-fixed-width"><hr class="u-no-margin--bottom"></div>
+  <div class="u-fixed-width">
     <h2>Publicise your snap</h2>
-    <div class="row">
-      <div class="col-4">
-        <h3 class="p-heading--4">
-          Snap Store buttons
-        </h3>
-        <p>
-          Use the “Get it from the snap store” badge on your marketing communications and pages to let your audience know about your snap. You can choose between a light and a dark button.
-        </p>
-      </div>
-      <div class="col-4">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/88f33387-04+Snap+Store+buttons+x2.png",
-            alt="",
-            width="344",
-            height="208",
-            hi_def=True
-          ) | safe
-        }}
-      </div>
+  </div>
+</section>
+
+<section class="p-strip is-shallow u-no-padding--top">
+  <div class="row u-hide--small u-hide--medium">
+    <div class="col-9 col-start-large-4"><hr class="u-no-margin--bottom"></div>
+  </div>
+  <div class="row">
+    <div class="col-3 col-start-large-4">
+      <h3 class="p-heading--5">Snap store buttons</h3>
+    </div>
+    <div class="col-6">
+      <p>Use the “Get it from the snap store” badge on your marketing communications and pages to let your audience know about your snap. You can choose between a light and a dark button.</p>
+      {{
+        image(
+          url="https://snapcraft.io/static/images/badges/en/snap-store-black.svg",
+          alt="",
+          height="56",
+          width="182",
+          hi_def=True,
+          loading="lazy",
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep u-no-padding--top">
+  <div class="row">
+    <div class="col-9 col-start-large-4"><hr class="u-no-margin--bottom"></div>
+  </div>
+  <div class="row">
+    <div class="col-3 col-start-large-4">
+      <h3 class="p-heading--5">GitHub badges</h3>
     </div>
-    <div class="row">
-      <div class="col-4">
-        <h3 class="p-heading--4">
-          GitHub badges
-        </h3>
-        <p>
-          Use the snap store GitHub badges to let your users know that they are able to get your app from Snapcraft. When your snap is flagged as trending you can also add a badge that signals this to your users.
-        </p>
-      </div>
-      <div class="col-4">
-        {{
-          image(
-          url="https://assets.ubuntu.com/v1/5f95707d-05+GitHub+badges+x2.png",
-            alt="",
-            width="344",
-            height="208",
-            hi_def=True
-          ) | safe
-        }}
-      </div>
+    <div class="col-6">
+      <p>Use the snap store GitHub badges to let your users know that they are able to get your app from Snapcraft. When your snap is flagged as trending you can also add a badge that signals this to your users.</p>
+      {{
+        image(
+          url="https://snapcraft.io/code/badge.svg",
```

Comment:
Can we upload the raw badge to the asset-server - we track the metrics of network loads of badges to get a rough idea where badges are around the internet, this will knock the `/code` badge out of reality.

Reply:
@Lukewh Done